translations/base-fr.yaml

```diff
@@ -722,11 +723,13 @@ settings:
                 fast: Rapide
                 super_fast: Très rapide
                 extremely_fast: Extrêmement rapide
+                
```

Comment:
You need to remove all the spaces in a free line. This also applies to lines 732, 749, 754, 763, 769, 775, 780, 785 and 790. You can look this up from the error messages in the CI check